src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -712,11 +766,18 @@ public void setMetadata() {
                 : TransactionViewModel.FILLED_SLOT;
     }
 
-    public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapshot, final LinkedHashSet<Hash> analyzedHashes)
-            throws Exception {
+    /**
+     * Update solid transactions
+     * @param tangle Tangle
+     * @param initialSnapshot Initial snapshot
+     * @param analyzedHashes analyzed hashes
+     * @throws Exception Exception
+     */
+    public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapshot,
+            final Set<Hash> analyzedHashes) throws Exception {
         Object[] hashes = analyzedHashes.toArray();
         TransactionViewModel transactionViewModel;
-        for(int i = hashes.length -1; i >= 0; i--){
+        for (int i = hashes.length - 1; i >= 0; i--) {
```

Comment:
usually it is best that formatting fixes will be in separate PRS

You can still leave it here for now